.github/workflows/testing.yaml

```diff
@@ -44,5 +44,5 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true'
 
       - name: Run chart-testing (install)
-        run: ct install --chart-dirs charts/velocity-operator --charts charts/velocity-operator --helm-extra-set-args "--set controllerManager.manager.env.dryRun=true"
-        if: steps.list-changed.outputs.changed == 'true'
\ No newline at end of file
+        run: ct install --upgrade --debug --chart-dirs charts/velocity-operator --charts charts/velocity-operator --helm-extra-set-args "--set controllerManager.manager.env.dryRun=true" --target-branch ${{ github.event.repository.default_branch }}
```

Comment:
Curious: which branch would it select this far?

Reply:
`master` which doesn't exist. I think it is only used when `--upgrade` flag is enabled so no use until now